extensions/amp-video/0.1/flexible-bitrate.js

```diff
@@ -39,6 +39,9 @@ const BITRATE_BY_EFFECTIVE_TYPE = {
 /** @const {number} Do not downgrade the quality of a video that has loaded enough content */
 const BUFFERED_THRESHOLD_PERCENTAGE = 0.8;
 
+/** @const {string} Simulates video being buffered (fully loaded) for the bitrate algorithm. */
+const VIDEO_BUFFERED_DEBUG_ATTR = 'i-amphtml-video-buffered-debug';
```

Comment:
Let's make it very clear what it does and what it's used for
eg: `IS_VIDEO_FULLY_LOADED_OVERRIDE_FOR_TESTING = i-amphtml-is-video-fully-loaded-override-for-testing`